.brigade/brigade.ts

```diff
@@ -141,7 +141,6 @@ const publishChartJob = (event: Event, version: string) => {
     "HELM_PASSWORD": event.project.secrets.helmPassword
   })
 }
-jobs[publishChartJobName] = publishChartJob
```

Comment:
The jobs map exists to facilitate re-execution of individual check_runs. That is not a concern for this job as this job is only run at release time.